### PR TITLE
Forms

### PR DESCRIPTION
**Pull request by catherinemarston**
Updated event component - SCSS needs to be fixed. Timepicker needs to be added on event component as well. Updated event-card-component to be a new template as per Alex's suggestions.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
     "@angular/compiler": "2.0.0-rc.5",
     "@angular/compiler-cli": "0.6.0",
     "@angular/core": "2.0.0-rc.5",
-    "@angular/forms": "2.0.0-rc.6",
+    "@angular/forms": "2.0.0-rc.5",
```
Comment (rodneystrong) [inline package.json:19]: @catherinemarston 

could you explain why rc5 would be better than rc6 please?

**Comment by rodneystrong** [comment]
@catherinemarston good effort, unfortunately your code for the event-card component isn't supposed to contain form fields since that component only displays completed cards (and doesn't create them). Your event component has good form logic though; let's pair and refactor a bit of this tomorrow.

Also have a few questions about the form logic and the decision to go from rc5 to rc6 on the form package. 

Let's discuss tomorrow with @smrtsmrf  after our standup.

**Comment by catherinemarston** [comment]
I asked you for clarification earlier about the event card component and if it was the component to create a template - there must have been some confusion there. We can just refactor it then for creating a new template. What are we calling that component?

I'm not sure why it went to rc5 forms - that was unintentional and it should definitely be rc6. 

Thanks!

**Comment by rodneystrong** [comment]
Unfortunately this pull request was made before the decision to go with template forms rather than model forms + the number of files changed (36) makes it difficult to go through all the conflicts to then safely merge. Closing this in favor of more recent code.

**Pull request closed without merging**